### ISSUE TITLE
Can't open after last update

### ISSUE THREAD

**Issue by YeldhamDev** [flathub/org.shadered.SHADERed#6]
```
shadered: error while loading shared libraries: libSpvGenTwoLib.so: cannot open shared object file: No such file or directory
```

**Comment by Calinou** [comment]
Maybe the CMake installation script isn't installing one of the required libraries? cc @dfranx

**Comment by stevewells20** [comment]
I'm also getting this error after installing SHADERed with flatpak...

**Comment by Calinou** [comment]
I can confirm the latest version (built with `flatpak-builder`) didn't work locally.

I reverted the upgrade to 1.3.6 until we find a way to fix it. The Flatpak available on Flathub should work again in a few hours.

**Comment by Calinou** [comment]
Closing, as this was resolved by downgrading to 1.3.5. See #10 for a pull request that updates SHADERed to the latest version.
